packages/react-hooks/src/mixer/useMixer.ts

```diff
@@ -96,7 +84,7 @@ export const useMixer = () => {
     try {
       let gens = await generateBulletproof();
       logger.trace(`Bulletproof `, gens.length);
-      const mixer = await Mixer.init(new Worker(), mixerIds, gens);
+      const mixer = await Mixer.init(new Worker(), mixerIds.assetGroups(), gens);
```

Comment:
Are we intending to initialize the mixer with all the groups? This seems contrary to what we discussed a few days back as far as initializing ONLY one asset group (currency, amount) pair.

Since we are polling for only 1 mixer info, shouldn't we also reflect initializing the mixer with only 1 group as well?

Reply:
Yeah this will be in a next pr